doc/classes/OS.xml

```diff
@@ -635,7 +635,7 @@
 		<method name="open_midi_inputs">
 			<return type="void" />
 			<description>
-				Initializes the singleton for the system MIDI driver, allowing Godot to receive [InputEventMIDI]. See also [method get_connected_midi_inputs] and [method close_midi_inputs].
```

Comment:
I'm not sure why this text was removed? It seems like helpful information.

Reply:
Oops. Small regression from my prior PR which added exactly that.